about.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About Us</title>
+  <link rel="stylesheet" href="saas/styles.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet">
+
+</head>
+<body>
+  <header >
+    <nav class="navbar">
+
+     <div class="hambuger">
+      <span class="bar"></span>
+      <span class="bar"></span>
+      <span class="bar"></span>
+     </div>
+
+     <ul class="nav-menu">
+      <li class="nav-items"><a href="index.html">Home</a></li>
+      <li class="nav-items"><a href="about.html" >About Us</a></li>
+      <li class="nav-items"><a href="#Courses" >Courses</a></li>
+      <li class="nav-items"><a href="#Contact" >Contact</a></li>
+    </ul>
+    </nav>
+
+    <div class="header-1">
+      <nav class="header-1d">
+        <ul>
+          <li><a href=""></a><i class="fa fa-facebook"></i></li>
+          <li><a href=""></a><i class="fa fa-twitter"></i></li>
+          <li><a href=""></a>English</li>
+          <li><a href=""></a>My Page Layout</li>
+          <li><a href=""></a>Logout</li>
+        </ul>
+      </nav>
+    </div>
+    <div class="header-2">
+      <nav class="desk-nav">
+        <div class="logo1"> <img src="./pics/partners/cck-logo1 1.png"></div>
+        <ul class="nav-d">
+          <li class="item"><a href="#Aboutus">About</a></li>
+          <li class="item"><a href="index.html">Program</a></li>
+          <li class="item"><a href="#">Join</a></li>
+          <li class="item"><a href="#">Sponsor</a></li>
+          <li class="item"><a href="#">Updates</a></li>
+        </ul>
+        <div class="logo2">Campaign</div>
+      </nav>
+    </div>
+  </header>
+
+  <main >
+   
+  <section class="Aboutus"><a name="Aboutus"></a>
+
+    <div class="about">
+
+      <div class="about-h">
+        <p> "Hello! Kid Developers" </p>
+        <h1> <strong>  INFORMATION <span style="color: rgb(134, 129, 36);"> TECHNOLOGY</span> FOR <span style="color: rgb(24, 105, 105);"> KIDS</span> </strong> </h1>
+      </div>
+      <div class="about-p">
+        <p>Technology is becoming more and more unimaginable. Why shouldn’t our children benefit from these blessings?
+           With tech at hand and close to everyone ,its time for children to discover their talent at an early stage. </p>
+      </div>
```

Comment:
Great work on your project! As mentioned by the previous reviewer, it's important to follow the design guidelines provided in the [design documentation](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) to ensure consistency and visual appeal. Please pay attention to the following aspects:

- Colors: Ensure that you use the specified color palette mentioned in the design documentation to maintain visual harmony.
- Typography: Make sure to use the recommended font face, size, and weight. The suggested font, [Lato](https://www.latofonts.com/), is a free font that closely resembles the one used in the original design.
- Layout: Pay attention to the composition and spacing between elements. Make sure they match the design specifications to achieve the desired visual balance.
![image](https://github.com/hydaline-code/Capstone_project_1/assets/55840999/c67c4647-124b-4f86-824b-76d1fdec1dd6)

Taking these guidelines into consideration will help your project closely resemble the original design and create a polished and cohesive look. Keep up the great work! 👍🌟